go/teams/transactions.go

```diff
@@ -642,6 +643,22 @@ func (tx *AddMemberTx) Post(ctx context.Context) (err error) {
 		if !found {
 			return fmt.Errorf("AddMemberTx.Post got a PerTeamKey but couldn't find a link with None to attach it")
 		}
+	} else { // If we didn't rotate the PTK, let's rebox the existing PTK for any of the new members
+		ekLib := g.GetEKLib()
+		if ekLib != nil {
+			// Only box up for the latest members
+			memSet.removeExistingMembers(ctx, team)
+			i := 0
+			uids := make([]keybase1.UID, len(memSet.recipients))
+			for uv := range memSet.recipients {
```

Comment:
A team can contain reset incarnations of users. You don't want to box for them.

Reply:
discussed offline: `removeExistingMembers` should handle this case